dnf-behave-tests/features/scriptlets.feature

```diff
@@ -92,3 +92,113 @@ Scenario: Remove a pkg with a failing %postun scriptlet
    And Transaction is following
        | Action        | Package                           |
        | remove        | Package-postun-fail-0:1.0-1.x86_64 |
+
+
+@bz1724779
+@not.with_os=rhel__eq__8
+Scenario: Output for triggered successful scriptlet of a package not present in transaction has temporarily just pkg name
+ Given I successfully execute dnf with args "install Package-triggerin-ok"
+  When I execute dnf with args "install Package-install-file"
+  Then the exit code is 0
+   And Transaction is following
+       | Action        | Package                             |
+       | install       | Package-install-file-0:1.0-1.x86_64 |
+   And stdout contains "triggerin scriptlet \(Package-triggerin-ok\) for Package-install-file install/update successfully done"
+   And stdout does not contain "Running scriptlet\s*:\s*Package-install-file"
+   And stdout contains "Running scriptlet\s*:\s*Package-triggerin-ok"
+   # We really should be matching the whole NEVRA of pkg but current RPM Python API provides 
+   # only the name (we can fix this by moving logic to libdnf, DNF5).
+   # This same situation is in all of the tests below.
+   # And stdout contains "Running scriptlet\s*:\s*Package-triggerin-ok-1.0-1.x86_64"
```

Comment:
I'm not sure how much it is a personal preference, but I'm used to comments going before the line to which they belong. Maybe just a nitpick, but seems to me it would actually be a bit clearer if they went before the last line of the scenario, which is what they are referencing I think (goes for all the comments at ends of scenarios in the file).

Reply:
Thank you. I think the comment line is indeed usually above. I moved them.